<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>220</height>
   </rect>
  </property>

  <widget class="QLabel" name="animLabel">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>25</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Animation Clip:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="categoryComboBox">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>20</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Name">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>20</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="animSearch">
   <property name="geometry">
    <rect>
     <x>741</x>
     <y>20</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="weightLabel">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>47</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Weight">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>45</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="cooldownLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>69</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cooldown Time (sec):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CooldownTime_Sec">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>67</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="moodLabel">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>93</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Mood:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Mood">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="headAngleLabel">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>123</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Head Angle:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="UseHeadAngle">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="headAngleMinLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>142</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Head Angle Min (deg):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HeadAngleMin_Deg">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>139</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="headAngleMaxLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>162</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Head Angle Max (deg):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HeadAngleMax_Deg">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>161</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="removeAnimLabel">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>192</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove this clip:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="removeAnimComboBox">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>189</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="confirmLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>192</y>
     <width>130</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Are you sure?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="confirmComboBox">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>188</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
